code/modules/ai/ai_holder_targeting.dm

```diff
@@ -129,6 +130,8 @@
 			if(L.stat == UNCONSCIOUS)	// Do we have mauling? Yes? Then maul people who are sleeping but not SSD
 				if(mauling)
 					return TRUE
+				else if(unconscious_vore && L.allowmobvore) //VOREStation Add
```

Comment:
Could this create an edge case where the mob will eat unconscious people, the prey allows mob vore, but something else prevents the mob from eating (such as being full) so it reverts to the standard mauling behaviour and claws them to death?

Reply:
That seems possible. I also was gonna look into what OOC escape does to make sure they don't maul people after they do that too.

Reply:
maybe return false if will_eat() fails or something?